exercises/circular-buffer/canonical-data.json

```diff
@@ -7,10 +7,10 @@
     "",
     "Tests will contain a number of operations. The operation will be specified in the `operation` key.",
     "Based on the operation, other keys may be present.",
-    "read: Reading from the buffer should produce the item located at `expected`.",
-    "write: Writing the item located at `item` should succeed.",
-    "readShouldFail: Reading from the buffer should fail.",
-    "writeShouldFail: Writing the item located at `item` should fail.",
+    "read: Reading from the buffer should succeed if and only if `should_succeed` is true.",
+    "  If it should succeed, it should produce the item at `expected`. ",
+    "  If it should fail, `expected` will not be present. ",
+    "write: Writing the item located at `item` should succeed if and only if `should_succeed` is true.",
```

Comment:
I added it just to be sure. I have a nagging feeling that if similar sentences appear in too many places...

* https://github.com/exercism/x-common/blob/master/exercises/largest-series-product/canonical-data.json#L3
* https://github.com/exercism/x-common/blob/master/exercises/hamming/canonical-data.json#L8
* https://github.com/exercism/x-common/blob/master/exercises/rna-transcription/canonical-data.json#L6
* https://github.com/exercism/x-common/blob/master/exercises/triangle/canonical-data.json#L15

then we might extract them all out to the contributing guide's section on porting an exercise. But I don't know what the right answer to that is yet, so for now I'll add the sentence.